main.go

```diff
@@ -1,13 +1,63 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/Percona-Lab/mnogo_exporter/exporter"
```

Comment:
please move it below as separate block

Reply:
Issues like that should break CI. Both goimports configuration and `git diff --exit-code` are missing